androidtv/setup_async.py

```diff
@@ -43,12 +43,14 @@ async def setup(host, port=5555, adbkey='', adb_server_ip='', adb_server_port=50
         atv = AndroidTVAsync(host, port, adbkey, adb_server_ip, adb_server_port, state_detection_rules, signer)
         await atv.adb_connect(auth_timeout_s=auth_timeout_s)
         atv.device_properties = await atv.get_device_properties()
+        atv.installed_apps = await atv.installed_apps()
```

Comment:
This will overwrite the `installed_apps()` method. You shouldn't assign new attributes outside `__init__()`. You could initialize it in `__init__()` and make `_installed_apps()` not be a static method and assign it there, or just return it and not assign it.